src/lib/utils/synapseTypes/DownloadListV2/ActionRequiredCount.ts

```diff
@@ -0,0 +1,22 @@
+// http://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/download/ActionRequiredCount.html
```

Comment:
Not yet used.  Introduced new types for future work (John is still working on the Actions Required services)